docs/check.md

```diff
@@ -1,71 +1,38 @@
-## About
+# Check
 
+## About
 This feature checks whether the commit message follows the given committing rules.
+If you want to setup an automatic check before every git commit, please refer to 
+[Automatically check message before commit](auto_check.md)
 
-## Checking before the commit
-To automatically check a commit message prior to committing, you can use a [git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks). There are two common methods for installing the hook:
-
-### Method 1: Add git hook through [pre-commit](https://pre-commit.com/)
-
-* Step 1: Install [pre-commit](https://pre-commit.com/)
-
-```sh
-python -m pip install pre-commit
-```
-
-* Step 2: Create `.pre-commit-config.yaml` at your root directory with the following content
+## Usage
+There are three arguments that you can use one of them to check commit message
 
-```yaml
----
-repos:
-  - repo: https://github.com/commitizen-tools/commitizen
-    rev: v1.17.0
-    hooks:
-      - id: commitizen
-        stages: [commit-msg]
+### Commit Message File
+```bash
+$ cz check --commit-msg-file COMMIT_MSG_FILE
 ```
+In this option, COMMIT_MSG_FILE is the path of the temporal file that contains the commit message
```

Comment:
I'd suggest move this session to the lastest sesssion. Also, we can metion that it's useful when cooperate with git hook. Otherwise, I doubt many would check commit message in a file.